### PR TITLE
fix(reg): Failed to load x:Load tagged elements using xBind visibility expression

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

x:Bind Visibility on x:Load tagged elements fail to load the elements.

## What is the new behavior?

x:Load elements are loaded properly and observe their source changes.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs [hunk 1]
```diff
@@ -4827,7 +4827,7 @@ private IDisposable TryGenerateDeferedLoadStrategy(IIndentedStringBuilder writer
 				var dataContextMember = FindMember(definition, "DataContext");
 				var nameMember = FindMember(definition, "Name");
 
-				if (!targetType.Is(_elementStubSymbol.BaseType))
+				if (!(targetType?.Is(_elementStubSymbol.BaseType) ?? false))
```
Comment (davidjohnoliver) [inline src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs:4830]: what no `#nullable enable`? 😝

File: src/Uno.UI/UI/Xaml/ElementStub.Android.cs [hunk 2]
```diff
@@ -22,12 +22,15 @@ private View MaterializeContent()
 
 			if (currentPosition != null && currentPosition.Value != -1)
 			{
+				// Create the instance first so that x:Bind constructs can be picked up by the
+				// Unload event of ElementStub. Not doing so does not fills up the generated variables
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/ElementStub.Android.cs:26]:
```suggestion
				// Unload event of ElementStub. Not doing so fills up the generated variables
```

File: src/Uno.UI/UI/Xaml/ElementStub.iOSmacOS.cs [hunk 3]
```diff
@@ -25,11 +25,14 @@ private UIView MaterializeContent()
 			
 			if (currentPosition != -1)
 			{
+				// Create the instance first so that x:Bind constructs can be picked up by the
+				// Unload event of ElementStub. Not doing so does not fills up the generated variables
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/ElementStub.iOSmacOS.cs:29]:
```suggestion
				// Unload event of ElementStub. Not doing so fills up the generated variables
```

File: src/Uno.UI/UI/Xaml/ElementStub.netstd.cs [hunk 4]
```diff
@@ -22,11 +22,16 @@ private FrameworkElement MaterializeContent()
 
                 if (currentPosition != -1)
                 {
-                    parentElement.RemoveChild(this);
+					// Create the instance first so that x:Bind constructs can be picked up by the
+					// Unload event of ElementStub. Not doing so does not fills up the generated variables
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/ElementStub.netstd.cs:26]:
```suggestion
					// Unload event of ElementStub. Not doing so fills up the generated variables
```

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/ElementStub.netstd.cs (hunk 4 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/ElementStub.netstd.cs:26]: Fixing this in a different PR as this one is time sensitive

**Pull request merged**